src/ci/docker/host-x86_64/test-various/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

Comment:
Why is this necessary when we are already using `-y`?

Reply:
Avoids prompts while configuring packages -- specifically necessary for tzdata.